lib/stat_tracker.rb

```diff
@@ -39,10 +40,26 @@ def turn_game_teams_csv_data_into_game_teams_objects(game_teams_csv_data)
     game_teams_objects_collection
   end
 
-  def highest_total_score
-    output = @games.max_by do |game|
-      game.total_game_score
+
+  def total_games
+    games = []
+    @game_teams.map do |game|
+      games << game.result
+    end
+  end
+
+  def percentage_tie
+    game_ties = @game_teams.select do |game|
+      game.result == "TIE"
     end
-    output.total_game_score
+    (game_ties.count / total_games.count.to_f).round(2)
```

Comment:
Do we need the total_games method? I was thinking you could just call 
(game_ties.count / @game_teams.count.to_f).round(2) at the end of percentage_tie method.